rustfmt.toml

```diff
@@ -0,0 +1,4 @@
+max_width = 100
```

Comment:
I don't think we should repeat this default value (also mentioned in linebender/velato#13)

Reply:
Do you have an alternative suggestion? This is the value used in other linebender repos, e.g. glazier/peniko (if I recall correctly which repos those were)

Reply:
I would suggest not including it. I.e.
```suggestion
```

Reply:
It is included in the other repos indeed. However given that it is the default value, it doesn't really matter. I don't have strong feelings either way, so I guess let's just remove it.

Reply:
To clarify, I would like to remove it because it suggests we're choosing something other than the default

The line is doing nothing but increasing the potential confusion. I don't think it can have any advantage